docker-compose.yaml

```diff
@@ -38,6 +43,39 @@ services:
       - gamedb
       - tests
 
+  orchestrator:
+    build:
+      context: docker/gradle
+      target: dev
+      args:
+        - USER_ID=${USER_ID:-1000}
+        - GROUP_ID=${GROUP_ID:-1000}
+    user: '$USER_ID:$GROUP_ID'
+    volumes:
+      - .:/mnt/app
+      - orchestratorcache:/home/gradle/.gradle
+      - /mnt/app/.gradle
+    networks:
+      - gamedb
+      - orchestrator
+
+  login:
+    build:
+      context: docker/gradle
+      target: dev
+      args:
+        - USER_ID=${USER_ID:-1000}
+        - GROUP_ID=${GROUP_ID:-1000}
+    user: '$USER_ID:$GROUP_ID'
+    volumes:
+      - .:/mnt/app
+      - logincache:/home/gradle/.gradle
+      - /mnt/app/.gradle
+    networks:
+      - gamedb
+      - orchestrator
+      - login
+
```

Comment:
Not sure about this, maybe use the makefile to generate docker-compose run --rm commands ?